docs/plugins.rst

```diff
@@ -158,6 +158,18 @@ If there is a match, the user is added to the group with the specified access
 level.
 
 
+Download Statistics
+-------------------
+
+This plugin tracks and records file download activity. The recorded information,
```

Comment:
I'm not sure I agree :) 
Maybe I should use parentheses though. 
`The recorded information
(downloads started, downloads completed, and total requests made) is stored on the
file model:`